Assets/Scripts/Movement/ThirdPersonController.cs

```diff
@@ -56,5 +61,16 @@ void Update()
     {
         input.x = Input.GetAxisRaw("Horizontal");
         input.y = Input.GetAxisRaw("Vertical");
+
```

Comment:
I would use the mouse position in world space from your MouseLook script. You can draw a vector from the player to the mouse position and use that vector to lerp your rotation towards it